rhel8/profiles/cis.profile

```diff
@@ -902,7 +902,7 @@ selections:
 
     ### 6.1.8 Ensure permissions on /etc/group- are configured (Scored)
 
-    
+
```

Comment:
Nice catches these whitespace changes. But I usually recommend to not do them in unrelated commits. It might create conflicts or complicate backporting.

Reply:
Oh I did not notice that, I did not do it on purpose... probably my editor does that. Should I revert it somehow? What is actually being changed? It is hard to debug it for me here on GH.

Reply:
42ee623912c55f895bb95fe2c23cf044ba89affa is the commit witch does unrelated changes.